contracts/voting-verifier/src/query.rs

```diff
@@ -197,11 +198,13 @@ mod tests {
     fn message(id: u64) -> Message {
         Message {
             cc_id: CrossChainId {
-                chain: "source_chain".parse().unwrap(),
-                id: format!("id:{id}").parse().unwrap(),
+                chain: "source-chain".parse().unwrap(),
+                id: format!("id{TX_HASH_EVENT_INDEX_SEPARATOR}{id}")
+                    .parse()
+                    .unwrap(),
             },
             source_address: format!("source_address{id}").parse().unwrap(),
-            destination_chain: format!("destination_chain{id}").parse().unwrap(),
+            destination_chain: format!("destination-chain{id}").parse().unwrap(),
```

Comment:
why change this?

Reply:
we disallow underscore now, to match core